multicluster/service-mirror/cluster_watcher_mirroring_test.go

```diff
@@ -453,6 +590,199 @@ func TestRemoteEndpointsUpdatedMirroring(t *testing.T) {
 	}
 }
 
+/// Test behavior on the remote cluster adding endpoint for pod-1 on an exported global service
+func TestRemoteEndpointSlicesUpdatedMirroring(t *testing.T) {
```

Comment:
Is it possible to add a test where the global mirror name differs from the exported headless service? Just to have a sanity check that the logic still holds up. Do you think that would make sense?